### DISCUSSION TITLE
How can I install a Spanish spell checker for quarto documents ?

### DISCUSSION THREAD

**Discussion by Aleinvarg** [quarto-dev/quarto-cli#7714] [Q&A]
### Description

I am transitioning to Quarto for my academic writing. I would like to know if it is possible to run a Spanish spell checker. The preinstalled English spell checking works great, but I haven't found a way to change the language of the checker.

**Comment by mcanouil** [comment]
Quarto is a command line tool, it does not have spell checker.
A spell checker is from the IDE you are using, so check from your IDE documentation.
